### DISCUSSION TITLE
Filter by file type

### DISCUSSION THREAD

**Discussion by ankit-sinha-18** [aahnik/tgcf#61] [Q&A]
I have checked and tested new release, it is working fine for me,both live and past features..

But i want to forward only media files, not text ,,,so what should be the configuration for forwarding only media files,not text

tgcf.config.yml

**Comment by aahnik** [comment]
Currently, only text is supported, will be adding file-based filtering within a week. Stay tuned, and help me reach 100 stars.

and thanks for your testing and feedback.

>  have checked and tested new release, it is working fine for me,both live and past features..

this line gives a lot of satisfaction. ❤️
